pom.xml

```diff
@@ -179,6 +185,12 @@
             <artifactId>workflow-basic-steps</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>19.0</version>
+        </dependency>
```

Comment:
Please don't add guava as a dependency, especially an older guava version like this.

If you absolutely need the capabilities provided by guava, please review https://www.jenkins.io/blog/2021/11/09/guava-upgrade/ and find a way to rely on the guava that is provided in Jenkins

Reply:
Thank you for your answer. 
When I opened this PR, Guava wasn't updated into Jenkins and it was an older version than 19.0. 
I just took the first version that worked with my changes (because 11.0.1 provided before was too old).
I no longer need this dependency now, I can use the new version provided by Jenkins.